modules/tests/shared/src/test/scala/coursier/test/ResolutionTests.scala

```diff
@@ -676,6 +697,27 @@ object ResolutionTests extends TestSuite {
         assert(res == expected)
       }
     }
+
+    'mergingTransitiveDeps{
```

Comment:
This test did fail when tested without the fix (`my-lib-1` would apparently be resolved _without_ a version number).

Reply:
Ouch…